config/server.json

```diff
@@ -1,3 +1,3 @@
 {
-  "url": "TO-BE-PROVIDED"
+  "url": "https://kbss.felk.cvut.cz/termit-server-dev"
```

Comment:
why this value goes to git?

Reply:
This is of course an accidental change. Will fix it in the morning.